### PR TITLE
ticdc: refine changefeed scheduler configuration description

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nongfushanquan — APPROVED** [review]

**Review by CharlesCheung96 — APPROVED** [review]

**Review by nongfushanquan — COMMENTED** [review]
是否拆分事务的参数这个文档里面漏了

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ aliases: ['/docs-cn/dev/sync-diff-inspector/sync-diff-inspector-overview/','/doc
 * 如果数据不一致，则生成用于修复数据的 SQL 语句
 * 支持[不同库名或表名的数据校验](/sync-diff-inspector/route-diff.md)
 * 支持[分库分表场景下的数据校验](/sync-diff-inspector/shard-diff.md)
-* 支持 [TiDB 主从集群的数据校验](/sync-diff-inspector/upstream-downstream-diff.md)
+* 支持 [TiDB 主从集群的数据校验](/ticdc/ticdc-upstream-downstream-check.md)
```
Comment (ran-huang) [inline sync-diff-inspector/sync-diff-inspector-overview.md:16]: sync diff inspector 不支持这个功能了，还是说我们更推荐使用 cdc ？

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -101,22 +101,19 @@ rules = ['*.*', '!test.*']
...
+# 将表以 Region 为单位分配给多个 TiCDC 节点进行同步。
 # 注意：该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
 # 默认为 "false"。设置为 "true" 以打开该功能。
 enable-table-across-nodes = false
-
-# 打开该功能后，该功能会对 Region 个数大于 `region-threshold` 值的表生效。
-region-threshold = 100000
-
-# 打开该功能后，该功能会对每分钟修改行数大于 `write-key-threshold` 值的表生效。
+# 有两种分配模式
```
Comment (ran-huang) [inline ticdc/ticdc-changefeed-config.md:108]:
```suggestion
# enable-table-across-nodes 开启后，有两种分配模式
```

File: ticdc/ticdc-changefeed-config.md [hunk 3]
```diff
@@ -101,22 +101,19 @@ rules = ['*.*', '!test.*']
...
+# 1. 按 Region 的数量分配，即每个 CDC 节点处理 region 的个数基本相等 。当某个表 Region 个数大于 `region-threshold` 值时，会将表分配到多个节点处理。`region-threshold` 默认值为 10000。
```
Comment (ran-huang) [inline ticdc/ticdc-changefeed-config.md:109]:
```suggestion
# 1. 按 Region 的数量分配，即每个 CDC 节点处理 region 的个数基本相等。当某个表 Region 个数大于 `region-threshold` 值时，会将表分配到多个节点处理。`region-threshold` 默认值为 10000。
```

File: ticdc/ticdc-changefeed-config.md [hunk 4]
```diff
@@ -101,22 +101,19 @@ rules = ['*.*', '!test.*']
...
+# region-threshold = 10000 
+# 2. 按写入的流量分配，即每个 CDC 节点处理 region 总修改行数基本相当。当表中每分钟修改行数大于 `write-key-threshold` 值的表生效。
```
Comment (ran-huang) [inline ticdc/ticdc-changefeed-config.md:111]:
```suggestion
# 2. 按写入的流量分配，即每个 CDC 节点处理 region 总修改行数基本相当。只有当表中每分钟修改行数超过 `write-key-threshold` 值时，该表才会生效。
```

File: ticdc/ticdc-changefeed-config.md [hunk 5]
```diff
@@ -101,22 +101,19 @@ rules = ['*.*', '!test.*']
...
+# write-key-threshold = 30000
 # 注意：
-# * `write-key-threshold` 参数默认值为 0，代表该功能默认不会按表的修改行数来切分表的同步范围。
-# * 你可以根据集群负载来配置该参数，如 30000，代表当表每分钟的更新行数超过 30000 时，该功能将会切分表的同步范围。
-# * 当 `region-threshold` 和 `write-key-threshold` 同时配置时，
-#   TiCDC 将优先检查修改行数是否大于 `write-key-threshold`，
-#   如果不超过，则再检查 Region 个数是否大于 `region-threshold`。
-write-key-threshold = 0
+# `write-key-threshold` 参数默认值为 0，代表默认不会采用流量的分配模式。
+# 两种方式配置一种即可生效，当  `region-threshold` 和 `write-key-threshold` 同时配置时，TiCDC 将优先采用按流量分配的模式。
```
Comment (ran-huang) [inline ticdc/ticdc-changefeed-config.md:115]:
```suggestion
# 两种方式配置一种即可生效，当 `region-threshold` 和 `write-key-threshold` 同时配置时，TiCDC 将优先采用按流量分配的模式，即 `write-key-threshold`。      
```

**Review by sdojjy — APPROVED** [review]

File: ticdc/ticdc-upstream-downstream-check.md [hunk 6]
```diff
@@ -1,17 +1,21 @@
 ---
-title: TiDB 主从集群的数据校验
+title: 主从集群一致性读和数据校验
 aliases: ['/docs-cn/dev/sync-diff-inspector/upstream-downstream-diff/','/docs-cn/dev/reference/tools/sync-diff-inspector/tidb-diff/']
```
Comment (ran-huang) [inline ticdc/ticdc-upstream-downstream-check.md:3]:
```suggestion
aliases: ['/docs-cn/dev/sync-diff-inspector/upstream-downstream-diff/','/docs-cn/dev/reference/tools/sync-diff-inspector/tidb-diff/', '/zh/tidb/dev/upstream-downstream-diff']
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 7]
```diff
@@ -1,17 +1,21 @@
...
-# TiDB 主从集群的数据校验
+# TiDB 主从集群数据校验和快照读
 
-当你使用 TiCDC 搭建 TiDB 的主从集群时，可能会需要在不停止同步的情况下对上下游的数据进行一致性验证。在普通的同步模式中，TiCDC 只提供数据的最终一致性的保证，而无法确保在同步的过程中数据的一致性。因此，对动态变更的数据进行一致性校验非常困难，为了满足这一需求，TiCDC 提供了 Syncpoint 功能。
+当你使用 TiCDC 搭建 TiDB 的主从集群时，可能会需要在不停止同步的情况下对上下游进行一致性的快照读或者对数据进行一致性验证。在普通的同步模式中，TiCDC 只提供数据的最终一致性的保证，而无法确保在同步的过程中数据的一致性。因此，对动态变更的数据进行一致性读非常困难，为了满足这一需求，TiCDC 提供了 Syncpoint 功能。
 
 Syncpoint 通过利用 TiDB 提供的 snapshot 特性，让 TiCDC 在同步过程中维护了一个上下游具有一致性 snapshot 的 `ts-map`。把校验动态数据的一致性问题转化为了校验静态 snapshot 数据的一致性问题，达到了接近数据一致性实时校验的效果。
 
-要开启 SyncPoint 功能，你可以在创建同步任务时把 TiCDC 的配置项 `enable-sync-point` 设置为 `true`。开启 Syncpoint 功能后，TiCDC 在数据的同步过程中会根据你所配置的 TiCDC 参数 `sync-point-interval` 来定时对齐上下游的 snapshot，并将上下游的 TSO 对应关系保存在下游的 `tidb_cdc.syncpoint_v1` 表中。
+要开启 SyncPoint 功能，你可以在创建同步任务时把 TiCDC 的配置项 `enable-sync-point` 设置为 `true`。开启 Syncpoint 功能后，TiCDC 会向下游 TiDB 集群写入如下信息：
```
Comment (ran-huang) [inline ticdc/ticdc-upstream-downstream-check.md:12]:
```suggestion
要开启 SyncPoint 功能，只需在创建同步任务时把 TiCDC 的配置项 `enable-sync-point` 设置为 `true`。开启 Syncpoint 功能后，TiCDC 会向下游 TiDB 集群写入如下信息：
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 8]
```diff
@@ -1,17 +1,21 @@
...
 
-然后，你只需要在 sync-diff-inspector 中配置 `snapshot` 即可对 TiDB 主从集群的数据进行校验。以下 TiCDC 配置示例为创建的同步任务开启 Syncpoint 功能：
+1. TiCDC 在数据的同步过程中，会定期 (`sync-point-interval`) 对齐上下游的 snapshot，并将上下游的 TSO 对应关系保存在下游的 `tidb_cdc.syncpoint_v1` 表中。
+
+2. TiCDC 在同步过程中，会定期 (`sync-point-interval`) 在备用集群中通过执行 `SET GLOBAL tidb_external_ts = @@tidb_current_ts` 设置已复制完成的一致性快照点。
```
Comment (ran-huang) [inline ticdc/ticdc-upstream-downstream-check.md:16]:
```suggestion
1. 在数据的同步过程中，TiCDC 会定期（使用 `sync-point-interval` 参数配置）对齐上下游的快照，并将上下游的 TSO 的对应关系保存在下游的 `tidb_cdc.syncpoint_v1` 表中。

2. 同步过程中，TiCDC 还会定期（使用 `sync-point-interval` 参数配置）通过执行 `SET GLOBAL tidb_external_ts = @@tidb_current_ts` ，在备用集群中设置已复制完成的一致性快照点。
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 9]
```diff
@@ -1,17 +1,21 @@
...
+
+以下 TiCDC 配置示例为创建的同步任务开启 Syncpoint 功能：
```
Comment (ran-huang) [inline ticdc/ticdc-upstream-downstream-check.md:18]:
```suggestion
以下是 TiCDC 配置示例，用于在创建同步任务时启用 Syncpoint 功能：
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 10]
```diff
@@ -24,7 +28,17 @@ sync-point-interval = "5m"
...
-## 获取 ts-map
+## 一致性快照读
+
+当你需要从备用集群查询数据的时候，在业务应用中设置 `SET GLOBAL|SESSION tidb_enable_external_ts_read = ON;` 就可以在备用集群上获得事务状态完成的数据。
+
+除此之外，你也可以通过查询 `ts-map` 的方式选取之前的时间点进行快照读。
+
+## 数据一致性校验
+
+你只需要在 sync-diff-inspector 中配置 `snapshot` 即可对 TiDB 主从集群的数据进行校验。
```
Comment (ran-huang) [inline ticdc/ticdc-upstream-downstream-check.md:39]: 这里是 sync-diff-inspector 而不是 cdc 吗？

File: ticdc/ticdc-upstream-downstream-check.md
Comment (ran-huang) [inline ticdc/ticdc-upstream-downstream-check.md:1]: 【开启 syncpoint 功能】和【一致性快照读/数据一致性校验】的关系是什么？前者是使用后者的必要条件吗？

File: sync-diff-inspector/sync-diff-inspector-overview.md (hunk 1 above)
Comment (asddongmen) [reply sync-diff-inspector/sync-diff-inspector-overview.md:16]: 这个文档应该放在 cdc 里面而不是 syncdiff 里面，移动一下位置。

File: ticdc/ticdc-upstream-downstream-check.md
Comment (asddongmen) [reply ticdc/ticdc-upstream-downstream-check.md:1]: 是的。

File: ticdc/ticdc-upstream-downstream-check.md (hunk 10 above)
Comment (asddongmen) [reply ticdc/ticdc-upstream-downstream-check.md:39]: 是的

Comment (asddongmen) [reply ticdc/ticdc-upstream-downstream-check.md:39]: 要先使用 cdc 同步数据，才会使用 syncdiff 来进行数据一致性校验。

**Review by ran-huang — APPROVED** [review]

**Pull request merged**